server/cmwell-cons/src/main/scala/ComponentConf.scala

```diff
@@ -260,10 +260,10 @@ case class CassandraConf(home: String,
         "row_cache_size" -> rowCacheSize.toString,
         "cas_data_dirs" -> casDataDirs.map(dir=> s"$home/data/$dir/data").mkString("\n    - "),
         "concurrent_reads" -> (16 * casDataDirs.size).toString,
-        "concurrent_writes" -> (numOfCores / 2).toString,
+        "concurrent_writes" ->  (8 * Math.max(2, numOfCores / 2)).toString,
         "concurrent_counter_writes" -> (16 * casDataDirs.size).toString,
         "disk_optimization_strategy" -> diskOptimizationStrategy,
-        "concurrent_compactors" -> (if (diskOptimizationStrategy == "SSD") numOfCores / 2 else 1).toString
+        "concurrent_compactors" -> (if (diskOptimizationStrategy == "SSD") Math.max(1, numOfCores / 2) else 1).toString
       )
     )
```

Comment:
why do u have this code in your FB?this code is already pushed to master few weeks ago.
maybe you forget to reset our local changes in the debug session we had and pushed it now?

Reply:
I think I need to merge master...

Reply:
Fix pushed